src/utils/setFirebaseProvider.jsx

```diff
@@ -0,0 +1,22 @@
+import firebase from "../firebaseConfig";
```

Comment:
like i said before dont make a whole file for a single function. make it more general like authHelpers or something